releng/ide.target

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?pde version="3.8"?><target name="ELS Target" sequenceNumber="14">
+<locations>
+<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="com.google.gson" version="2.2.4.v201311231704"/>
+<unit id="org.json" version="1.0.0.v201011060100"/>
+<unit id="com.google.guava" version="15.0.0.v201403281430"/>
+<unit id="org.junit" version="4.12.0.v201504281640"/>
+<unit id="org.apache.commons.io" version="2.2.0.v201405211200"/>
+<repository location="http://archive.eclipse.org/tools/orbit/downloads/drops/R20160520211859/repository/"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="org.eclipse.platform.feature.group" version="4.6.0.v20160606-1342"/>
+<unit id="org.eclipse.wst.jsdt.feature.feature.group" version="2.0.0.v201605311817"/>
+<repository location="http://download.eclipse.org/releases/neon"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="org.eclipse.xtext.xbase.lib.feature.group" version="2.10.0.v201605250459"/>
+<repository location="http://download.eclipse.org/modeling/tmf/xtext/updates/composite/releases/"/>
+</location>
+<location path="${project_loc:eclipse-language-service}/deps-repository/target/repository/plugins" type="Directory"/>

```

Comment:
Yeah, that's good too, although user still needs to run build first, before setting the target platform (and needs to set M2_REPO env var if not set already). But this is all temporary anyways, we don't need a perfect solution now.